doc/examples/viz_hud.py

```diff
@@ -0,0 +1,146 @@
+"""
```

Comment:
Yes, HUD stands for Heads-Up Display (https://en.wikipedia.org/wiki/HUD_(video_gaming)).